pages/system/inputs.js

```diff
@@ -72,24 +80,36 @@ _handleChange = (e) => {
         <br />
         <System.Input
           label="Location of your pastries"
-          description="We need to know the location of your pastries to sell them to other people."
+          description="We need to know the location of your pastries."
           tooltip="Hey friends."
-          name="fourteen"
-          value={this.state.fourteen}
+          name="exampleTwo"
+          value={this.state.exampleTwo}
           placeholder="Pastry Location"
           onChange={this._handleChange}
         />
         <br />
         <System.CodeBlock>
-          {`<System.Input
-  label="Location of your pastries"
-  description="We need to know the location of your pastries to sell them to other people."
-  tooltip="Hey friends."
-  name="fourteen"
-  value={this.state.fourteen}
-  placeholder="Pastry Location"
-  onChange={this._handleChange}
-/>`}
+          {`class ExampleLabel extends React.Component {
+  state = { exampleTwo: null }
+
+  _handleChange = (e) => this._handleChange(
```

Comment:
This is not correct, you need

`_handleChange = e => this.setState({ [e.target.name]: e.target.value });`